src/components/ArticlePage.jsx

```diff
@@ -21,11 +21,23 @@ function ArticlePage() {
   const [isLoading, setIsLoading] = useState(false);
   const [value, setValue] = useState("Comments");
   const [article, setArticle] = useState({});
+  const [votes, setVotes] = useState();
 
   const handleChange = (event, newValue) => {
     setValue(newValue);
   };
 
+  console.log(article_id);
+
+  const handleUpVote = (event) => {
+    const article_id = article.article_id;
+    upVote(article_id);
+    fetchArticleById(article_id).then((article) => {
+      setVotes(article.votes);
+    });
```

Comment:
Why is fetchArticleById being used here? When making a patch request you should also be receiving the updated article already, this seems like a redundant request.